src/main/java/seedu/duke/utils/Parser.java

```diff
@@ -173,6 +173,19 @@ public static boolean isValidInputForCommand(String command, String[] arguments)
             }
             break;
         }
+        case UserCommandWord.SHIFT_MODULE_COMMAND: {
+            if (arguments.length != 2) {
```

Comment:
I would recommend adding static variables here like "shiftArgWordsNum" or something, to avoid using magic numbers!
<img width="491" alt="image" src="https://github.com/AY2324S1-CS2113-T17-4/tp/assets/110933149/8ad1190e-d97e-48ef-abbc-4f626c6e9e1e">